catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/databases/TableResourceTest.java

```diff
@@ -1052,6 +1052,55 @@ void delete_table_as_non_admin_401(TestInfo test) throws HttpResponseException {
     assertResponse(exception, FORBIDDEN, "Principal: CatalogPrincipal{name='test'} is not admin");
   }
 
+  @Test
+  void patch_tableAttributes_with_rbac(TestInfo test) throws IOException {
```

Comment:
Need to move this test to EntityResourceTest if following conditions are met:
- this draft PR is looking good directionally 
- we enable for all PATCH APIs for entities